crates/bevy_ecs/src/query/mod.rs

```diff
@@ -57,100 +57,6 @@ mod tests {
 
     #[test]
     fn query_filtered_exactsizeiterator_len() {
```

Comment:
Note that this test was redundant with `query_filtered_combination_size` because we merged the two, and is why it's deleted.